cabal.project

```diff
@@ -17,4 +17,6 @@ packages:
    gen/
    test/
    amazonka/
+
+optional-packages:
```

Comment:
`cabal` will error if a glob fails to match, which it does if you delete every generated package.

Reply:
Makes sense 👍🏽